ClearlyEnergy/django-helpdesk
# Saved search by organization

#### What's this PR do?
Adds an organization field to the SavedSearch model that allows for a user's saved searches to be filtered by their active organization. At time of migration, each user's saved searches will be copied to all of their organizations. They can then later delete searches that are not relevant to other organizations. In the case of a reverse migration, saved searches with duplicate names in multiple organizations will be collapsed down to one SavedSearch instance.

Unfortunately, due to the how Django handles migrations, this needed to be 3 separate migrations... not 2, not 1... exactly 3 to avoid database trigger conflicts.

#### How should this be manually tested?
Switch between active organization in the "all tickets" page and ensure there is change in the saved searches.

#### What are the relevant tickets?
https://clearlyenergy-inc.monday.com/boards/7029256041/pulses/7127745638
